builds/install/misc/replication.conf

```diff
@@ -0,0 +1,149 @@
+	#
+	# include_filter = 
+
+	# Pattern (regular expression) that defines what tables must be excluded from
+	# replication. By default, all tables are replicated.
+	#
+	# exclude_filter = 
+
+	# Directory to store replication log files.
+	#
+	# log_directory = 
+
+	# Prefix for replication log file names. It will be automatically suffixed
+	# with an ordinal sequential number. If not specified, database filename
+	# (without path) is used as a prefix.
+	#
+	# log_file_prefix
+
+	# Maximum allowed size for a single replication segment.
+	#
+	# log_segment_size = 16777216 # 16MB
+
+	# Maximum allowed number of full replication segments. Once this limit is reached,
+	# the replication process is temporarily delayed to allow the archiving to catch up. 
+ 	# If any of the full segments is not archived during one minute,
+ 	# the replication fails with an error.
+ 	#
+	# Zero means an unlimited number of segments pending archiving.
+	#
+	# log_segment_count = 8
+
+	# Delay, in milliseconds, to wait before the changes are synchronously flushed
+	# to the log (usually at commit time). This allows multiple concurrently committing
+	# transactions to amortise I/O costs by sharing a single flush operation.
+	#
+	# Zero means no delay, i.e. "group flushing" is disabled.
+	#
+	# log_group_flush_delay = 0
+
+	# Directory for the archived log files.
+	#
+	# Directory to store archived replication segments.
+	# It also defines the $(archpathname) substitution macro (see below).
+	#
+	# log_archive_directory = 
+
+	# Program (complete command line with arguments) that is executed when some
+	# replication segment gets full and needs archiving.
+	#
+	# This program MUST return zero ONLY if archiving has been performed successfully.
+	# In particular, it MUST return non-zero if the target archive already exists.
+	#
+	# Special predefined macros are available:
+	#   $(logfilename) - file name (without path) of the log segment being archived
+	#   $(logpathname) - full path name of the log segment being archived
+	#     same as log_directory + $(logfilename)      
+	#   $(archpathname) - suggested full path name for the archived segment
+	#     same as log_archive_directory + $(logfilename)      
+	#
+	# Simplest configuration is to use standard OS commands for archiving, e.g.:
+	#
+	# Linux: "test ! -f $(archpathname) && cp $(logpathname) $(archpathname)"
+	# or
+	# Windows: "copy $(logpathname) $(archpathname)"
+	#
+	# log_archive_command = 
+
+	# Timeout, in seconds, to wait until incomplete segment is scheduled for archiving.
+	# It allows to minimize the replication gap if the database is modified rarely.
+	#
+	# Zero means no intermediate archiving, i.e. segments are archived only after
+	# reaching their maximum size (defined by log_segment_size).
+ 	#
+	# log_archive_timeout = 60
+
+	# Connection string to the replica database (used for synchronous replication only).
+	# Expected format:
+	#
+	# [<login>:<password>@]<database connection string>
```

Comment:
What about login and password with "special" characters as `:` and `@`?

Reply:
Currently, you may omit both and use ISC_USER/ISC_PASSWORD instead. I can also add support for the quotes, i.e. "myn@me":"my:pwd". But speaking honestly, I'm not satisfied with the current solution, maybe you guys will raise some clever idea.

Reply:
Hmm... is it setting with value only, no name ?

Reply:
Ah, i see it below. It is "sync_replica".
Why not break it into 3 parts: login\password\connection string ?

Reply:
@hvlad, do you mean three different settings? How to define multiple replicas in this case? Now you can specify:
sync_replica user1:pwd1@/my/first/replica
sync_replica user2:pwd2@/my/second/replica
sync_replica user3:pwd3@/my/third/replica
This may be not very elegant, but with three settings per each replica it's going to be a mess.

Reply:
Just as idea, maybe we could use nested sections:
database = /my/db
{
  sync_replica = /my/first/replica
  {
    user = user1
    password = pwd1
    ## may be other connection options
  }
}
but I don't know whether such nesting is supported by our config classes.